lib/main.dart

```diff
@@ -1,4 +1,5 @@
 import 'package:flutter/material.dart';
```

Comment:
Remove this, main.dart, home property should be splash screen

Reply:
ok